classes/mender-install.bbclass

```diff
@@ -13,11 +6,4 @@ VIRTUAL-RUNTIME_init_manager = "systemd"
 DISTRO_FEATURES_BACKFILL_CONSIDERED = "sysvinit"
 VIRTUAL-RUNTIME_initscripts = ""
 
-
-#Make sure we are creating sdimg with all needed partitioning.
-IMAGE_CLASSES += "sdimg"
-IMAGE_FSTYPES += "sdimg"
-
-IMAGE_INSTALL_append = " u-boot-fw-utils mender ca-certificates"
-
-
+IMAGE_INSTALL_append = " mender ca-certificates"
```

Comment:
[Done](https://tracker.mender.io/browse/MEN-448)! Feel free to add more clarification to it.